### PR TITLE
feat: get contract addresses for multisigs that need to be watched in config

### PR DESCRIPTION
**Pull request by nickreynolds**
- also publish event on multisig found to match newsroom found event

### CODE REVIEW

File: pkg/eventcollector/listener.go [hunk 1]
```diff
@@ -292,16 +292,21 @@ func (c *EventCollector) handleEvent(payload interface{}) interface{} {
 	}
 
 	// We need to get past newsroom events for the newsroom contract of a newly added watcher
-	if event.EventType() == "Application" {
-		newsroomAddr := event.EventPayload()["ListingAddress"].(common.Address)
+	if event.EventType() == "Application" || (event.ContractName() == "NewsroomFactory" && event.EventType() == "Instantiation") {
```
Comment (petersng) [inline pkg/eventcollector/listener.go]: Minor and prob can be done later, but these little checks could be wrapped in helper methods to make this more concise and readable.

Ex:
```
if c.isTcrApplicationEvent(event) || c.isNewsroomFactoryInstantiationEvent(event) {
    ...
}

...

func (...) isNewsroomFactoryInstantiationEvent(event ...) bool {
     return (event.ContractName() == "NewsroomFactory" && event.EventType() == "Instantiation")
}

func (...) isTcrApplicationEvent(event ...) bool {
     return (event.ContractName() == "CivilTCR" && event.EventType() == "Application")
}

```

File: pkg/eventcollector/listener.go [hunk 2]
```diff
@@ -312,6 +317,32 @@ func (c *EventCollector) handleEvent(payload interface{}) interface{} {
 		}
 	}
 
+	// We need to get past newsroom events for the newsroom contract of a newly added watcher
+	if event.EventType() == "Application" || (event.ContractName() == "MultiSigWalletFactoryContract" && event.EventType() == "Instantiation") {
```
Comment (petersng) [inline pkg/eventcollector/listener.go]: Same as above.

File: pkg/pubsub/pubsub.go [hunk 3]
```diff
@@ -23,6 +23,7 @@ func NewCrawlerPubSub(projectID string, topic string) (*CrawlerPubSub, error) {
 // CrawlerPubSubMessage is the message sent from the crawler
 type CrawlerPubSubMessage struct {
 	NewsroomException bool   `json:"newsroomException"`
+	MultiSigException bool   `json:"multiSigException"`
```
Comment (petersng) [inline pkg/pubsub/pubsub.go]: Didn't see it yet, but is there code in the processor to handle this?

File: pkg/utils/config_test.go [hunk 4]
```diff
@@ -302,8 +261,8 @@ func TestFetchListingAddresses(t *testing.T) {
 		t.Errorf("Should have fetched listing addresses: len: %v, err: %v",
 			len(config.ContractAddresses["newsroom"]), err)
 	}
-	// 1 newsroom from config, 14 from graphql
-	if len(config.ContractAddressObjs["newsroom"]) != 15 {
+	// 1 newsroom from config, 7 from graphql
+	if len(config.ContractAddressObjs["newsroom"]) != 8 {
```
Comment (petersng) [inline pkg/utils/config_test.go]: Need a test for the fetch multisig method?

File: pkg/pubsub/pubsub.go (hunk 3 above)
Comment (nickreynolds) [reply pkg/pubsub/pubsub.go]: going to add to processor as soon as this goes in

File: pkg/utils/config_test.go (hunk 4 above)
Comment (nickreynolds) [reply pkg/utils/config_test.go]: sure, i'll add one

**Pull request merged**